dockstore-integration-testing/src/test/java/io/dockstore/webservice/UserResourceIT.java

```diff
@@ -340,6 +340,29 @@ public void testAdminLevelSelfDestruct() {
         assertEquals(3, userCount);
     }
 
+    @Test
+    public void testGettingUserEmails() {
+        ApiClient client = getWebClient(OTHER_USERNAME, testingPostgres);
+        UsersApi userApi = new UsersApi(client);
+        ApiClient adminWebClient = getWebClient(ADMIN_USERNAME, testingPostgres);
+        UsersApi adminUserApi = new UsersApi(adminWebClient);
+
+        List<String> emails = adminUserApi.getAllUserEmails();
+        assertTrue(emails.isEmpty());
+
+        testingPostgres.runUpdateStatement("UPDATE user_profile set email = 'fakeEmail@example.com'");
+        emails = adminUserApi.getAllUserEmails();
+        assertEquals(2, emails.size());
```

Comment:
Maybe `>2` 
This seems like it will break when we add a user to the testing database which seems inevitable